soh/soh/GameMenuBar.cpp

```diff
@@ -449,6 +449,8 @@ namespace GameMenuBar {
         CVar_SetS32("gDayGravePull", 1);
         // Pull out Ocarina to Summon Scarecrow
         CVar_SetS32("gSkipScarecrow", 1);
+        // Chest size & texture matches contents
+        CVar_SetS32("gChestSizeAndTextureMatchesContents", 1);
```

Comment:
is there a way we could split these up? i really like the texture matching contents, but having giant chests in small chest locations looks a bit odd to me. it'd be nice to be able to have the textures match contents while keeping the original chest size

Reply:
Are you proposing like a dropdown instead of a checkbox with "Off", "Both", "Sizes only" "Textures only" or what?

Reply:
done here